eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/group/ClientGroupWrapper.java

```diff
@@ -486,6 +486,7 @@ public synchronized void initClientGroupPersistentConsumer() throws Exception {
                                 group, topic, bizSeqNo, sendBackTimes,
                                 sendBackFromEventMeshIp);
 
+                        sendBackTimes = Objects.requireNonNull(sendBackTimes);
                         if (sendBackTimes >= eventMeshTCPServer
```

Comment:
you can add `Ojects.requireNoNull(sendBackTimes)` at line 490 --> `if(Ojects.requireNoNull(sendBackTimes) >= eventMeshTCPSever`